### DISCUSSION TITLE
Keymap Mode and Projectile not playing nice

### DISCUSSION THREAD

**Discussion by stmonty** [meow-edit/meow#430] [Q&A]
Hey, I've been using Meow for a few days, and I noticed that Projectile and Meow are not working together properly with my config.

Basically, whenever I use `SPC p`, it just writes a `p` in the buffer, as if it was in `insert-mode` and then it goes back to `normal-mode`.

Even weirder, it seems to mess with being able to write the letter `p` in the minibuffer. For example, if I write `M-x` and then I try to write a `p` (lets say trying to write something like `M-x python-mode`) when I try to put the letter p in the minibuffer, my Emacs seems to think that I am trying to call the projectile command map! This seems to be an issue with any minibuffer writing, for example using `C-x b` and then trying to find a file that has a `p` in it will also do the same thing.

So I can't even write the letter p in the minibuffer now. But weirdly, this doesn't always happen. It seems like the longer I have Emacs open, the more likely this is to happen, as when I restart Emacs, I can write p in the minibuffer generally.

This doesn't seem to be a problem with any other mapping I have under `C-c` except `C-c p`.
Any ideas?

Here is the Projectile and Meow config:

```
;; Projectile
(use-package projectile
  :diminish projectile-mode
  :config (projectile-mode)
  :custom ((projectile-completion-system 'default))
  :bind (:map projectile-mode-map
  ("C-c p" . projectile-command-map))
  :init
  (when (file-directory-p "~/repos")
    (setq projectile-project-search-path '("~/repos")))
  (setq projectile-switch-project-action #'projectile-dired))

(defun meow-setup ()
   ;; Default Meow-Setup for qwerty keyboards as described in the Meow project repo

(use-package meow
  :config
  (meow-setup)
  (setq meow-keypad-describe-keymap-function nil)
  (meow-global-mode 1))

```

**Comment by DogLooksGood** [comment]
Try add this line:
```
(setq meow-keypad-leader-dispatch "C-c")
```
Then it should be able to find `C-c p` in any keymap.

**Reply by stmonty** [reply]
I added that to my `use-package meow` and now it just says that `C-c p` is undefined, which is strange.

**Reply by DogLooksGood** [reply]
Is it available with `C-c p`(without keypad)?

**Reply by stmonty** [reply]
> Is it available with `C-c p`(without keypad)?

No it isn't. `C-c p` doesn't work in general.

**Reply by DogLooksGood** [reply]
Then I guess there's something wrong with your `C-c p` key binding. It's unrelated to meow if it doesn't work in general.
